### PR TITLE
Corrects checking of total weights in the attribute file

### PR DESCRIPTION
**Pull request by vero-so**
### What have I made? 👩‍💻

Small PR that fixes the checking of presence of total weight in the attribute file was not correct. Processing failed only if the entire "Grand total" line was missing, but resulted in 0.0 total weight if the partial sums were just missing (probably due to wrong export template).

Added a guard for this case as well as a test file/test.

Related to AB#

### How can you best review this? 🧐

There is only little code, just have a look and see if it makes sense -- and if some other edge cases might still be forgotten.

### Did I remember everything checklist?

- [x] I made some awesome changes! 😎
- [x] I left the code in a better state than when I started working on it ✨
- [x] I wrote unit and/or integration tests 👾
- [ ] I have considered any security implications, and requested review of them explicitly 🔐
- [ ] I verified that it checks the acceptance criteria. 📜
- [ ] I have tested this change in the Echo DEV environment, or requested someone to test it for me 🚀
- [ ] I updated the documentation/readme 📚
- [x] I made the PR title descriptive and human-readable 👨‍👩‍👦‍👦
- [ ] Existing unreported issues I found but could not fix was added as a new Issues 🚧

### CODE REVIEW

File: CadRevealFbxProvider.Tests/TestSamples/attributes/missing_total_weight_entries.csv [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;19
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;20
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;21
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;22
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;23
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;24
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;25
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;26
```
Comment (Strepto) [inline CadRevealFbxProvider.Tests/TestSamples/attributes/missing_total_weight_entries.csv:28]: Consdier making hte file shorter. No need for all the data except one line and one Grand total

Comment (vero-so) [reply CadRevealFbxProvider.Tests/TestSamples/attributes/missing_total_weight_entries.csv]: done :)
